src/js/core.js

```diff
@@ -0,0 +1,126 @@
+var d3 = require('d3');
+var _ = require('underscore');
+var utils = require('./utils');
+
+function compute_svg_width(rect_width, rect_padding, row_length) {
```

Comment:
True, but I prefer code over comments. This function can also be moved to the utils file and unit tested. Yes, I need to setup some unit testing!